apps/api/.env.development

```diff
@@ -20,3 +20,10 @@ BUNDLE_ANALYZER=false
 PORT=3000
 # Default : 3443
 HTTPS_PORT=3443
+
+# - - - - - - - - - Database - - - - - - -
+SQL_SERVER_HOST=0.0.0.0
```

Comment:
Does have to be the 0.0.0.0 and listen on all network interfaces? Or is the loopback 127.0.0.1 enough for local dev? Out of curiosity.

Reply:
I haven't tested out with 127.0.0.1. both should work fine

Reply:
The thing is that on 0.0.0.0 all your network will be able to see your server running. Not that might be such a big issue. But was curious if it was a deliberate choice.